lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -156,64 +156,69 @@ def offensive_described_class?(node)
 
           return if nearest_described_class.equal?(node)
 
-          full_const_name(nearest_described_class) == full_const_name(node)
+          FullConstNameFinder.call(nearest_described_class) ==
+            FullConstNameFinder.call(node)
```

Comment:
This is not very self-explanatory when I read it aloud.

Reply:
I agree. Please review again.